README.md

```diff
@@ -94,64 +94,14 @@ twig:
 #### Required
 
 First, you have to select your language, other settings are optional (see below).
+If you don't have this file you can create one.
 
 ```yaml
-# config/packages/kms_froala_editor.yaml
+# config\packages\config.yaml 
```

Comment:
Not sure why the backslash instead of slash?
The docs are following the recipe & Symfony conventions here.
For separation of concerns & clarity, I think it's much easier to find configuration data in `kms_froala_editor.yaml` than a `config.yaml` file.

Reply:
@jmsche will undo the backslash changes